boards/xtensa/esp32s_nodemcu/doc/index.rst

```diff
@@ -0,0 +1,220 @@
+.. _esp32:
+
+ESP32
+#####
+
+Overview
+********
+
+ESP32 is a series of low cost, low power system on a chip microcontrollers
+with integrated Wi-Fi & dual-mode Bluetooth.  The ESP32S NodeMCU employs a
+Tensilica Xtensa LX6 dual-core microprocessor.  ESP32 is created and
+developed by Espressif Systems, a Shanghai-based Chinese company,
```

Comment:
ESP32S2 is not a LX6 but an LX7 Xtensa, it is also single core

Reply:
@uLipe This ESP32 based soc, not ESP32S2.

Reply:
Got tricked by the s in the board name. My bad